xdem/coreg/base.py

```diff
@@ -723,13 +711,28 @@ def __add__(self, other: CoregType) -> CoregPipeline:
             raise ValueError(f"Incompatible add type: {type(other)}. Expected 'Coreg' subclass")
         return CoregPipeline([self, other])
 
+    @property
```

Comment:
This was in the wrong spot (needs to be at `Coreg` level I think, not `Affine/NonAffine`!).